### PR TITLE
Docs: esm syntax

### PR DESCRIPTION
**Pull request by zekth**
Adding ESM syntax to examples, as its the modern way to write JS these days.
This covers the main readme and the getting started example. Not sure if the whole documentation needs a patch

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ yarn add fastify
 Let's write our first server:
 ```js
 // Require the framework and instantiate it
+
+// ESM
```
Comment (Eomm) [inline docs/Getting-Started.md:25]: Just to share, would it be more straightforward for users get a table

|esm|cjs|
|---|---|
code | code|

?

Comment (zekth) [reply docs/Getting-Started.md:25]: Won't this break the doc website? I will be hard to read on all devices

Comment (RafaelGSS) [reply docs/Getting-Started.md:25]: Just for curiosity, how the website would looks like with:

<details>
  <summary>ESM</summary>
  
  ```js
  import Fastify from 'fastify'
  const fastify = Fastify({
    logger: true
  })
```
</details>

<details>
  <summary>Commonjs</summary>
  
  ```js
  const fastify = require('fastify')({
    logger: true
  })
```
</details>

?

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**